### ISSUE TITLE
Jinja2Cpp build help needed - Bionic Beaver VM

### ISSUE THREAD

**Issue by DragonOsman** [flexferrum/Jinja2Cpp#65]
Hi.  I'm trying to build Jinja2Cpp on a Bionic Beaver VM.  I have CMake version 3.10 installed.  Here's the output for when I tried to build it:
```
/usr/local/jinja2cpp/build$ sudo cmake .. -DCMAKE_INSTALL_PREFIX=.
-- Configuring done
-- Generating done
-- Build files have been written to: /usr/local/jinja2cpp/build/thirdparty
[ 25%] Prebuild gtest library
CMake Error: The source directory "/usr/local/jinja2cpp/thirdparty/gtest" does not appear to contain CMakeLists.txt.
Specify --help for usage, or press the help button on the CMake GUI.
CMakeFiles/gtest.dir/build.make:68: recipe for target 'gtest/build/CMakeCache.txt' failed
make[3]: *** [gtest/build/CMakeCache.txt] Error 1
CMakeFiles/Makefile2:67: recipe for target 'CMakeFiles/gtest.dir/all' failed
make[2]: *** [CMakeFiles/gtest.dir/all] Error 2
CMakeFiles/Makefile2:79: recipe for target 'CMakeFiles/gtest.dir/rule' failed
Makefile:118: recipe for target 'gtest' failed
make[1]: *** [CMakeFiles/gtest.dir/rule] Error 2
make: *** [gtest] Error 2
CMake Error at CMakeLists.txt:83 (message):
  Can't build thirdparty libraries


-- Configuring incomplete, errors occurred!
See also "/usr/local/jinja2cpp/build/CMakeFiles/CMakeOutput.log".
See also "/usr/local/jinja2cpp/build/CMakeFiles/CMakeError.log".
dragonosman@Osman-VirtualBox:/usr/local/jinja2cpp/build$ sudo cmake --build . --target all
make: *** No rule to make target 'all'.  Stop.
dragonosman@Osman-VirtualBox:/usr/local/jinja2cpp/build$ sudo cmake --build .
make: *** No targets specified and no makefile found.  Stop.
dragonosman@Osman-VirtualBox:/usr/local/jinja2cpp/build$ ls thirdparty
CMakeCache.txt  CMakeFiles  cmake_install.cmake  gtest  Makefile  nonstd
dragonosman@Osman-VirtualBox:/usr/local/jinja2cpp/build$ cd thirdparty
dragonosman@Osman-VirtualBox:/usr/local/jinja2cpp/build/thirdparty$ ls
CMakeCache.txt  CMakeFiles  cmake_install.cmake  gtest  Makefile  nonstd
dragonosman@Osman-VirtualBox:/usr/local/jinja2cpp/build/thirdparty$ cd ..
dragonosman@Osman-VirtualBox:/usr/local/jinja2cpp/build$ cd ..
dragonosman@Osman-VirtualBox:/usr/local/jinja2cpp$ ls
appveyor.yml  cmake           include  README.md  test
build         CMakeLists.txt  LICENSE  src        thirdparty
dragonosman@Osman-VirtualBox:/usr/local/jinja2cpp$ cd build
dragonosman@Osman-VirtualBox:/usr/local/jinja2cpp/build$ ls
CMakeCache.txt  CMakeFiles  thirdparty
dragonosman@Osman-VirtualBox:/usr/local/jinja2cpp/build$ cmake --build .
make: *** No targets specified and no makefile found.  Stop.
dragonosman@Osman-VirtualBox:/usr/local/jinja2cpp/build$ cmake --build . --target install
make: *** No rule to make target 'install'.  Stop.
dragonosman@Osman-VirtualBox:/usr/local/jinja2cpp/build$ cmake --build . --target cmake_install.cmake
make: *** No rule to make target 'cmake_install.cmake'.  Stop.
```
I haven't done anything to the CMakeLists.txt file or anything else.  How do fix this problem?  Thanks in advance.

**Comment by flexferrum** [comment]
git submodule update --init

**Comment by flexferrum** [comment]
`.git` directory. Yes. You have to update submodules, which command I mention before does.

**Comment by DragonOsman** [comment]
When running the command 
```
cmake --build . --target install
```
I had this error message:
```
CMake Error at cmake_install.cmake:57 (file):
  file INSTALL cannot find
  "/usr/local/jinja2cpp/thirdparty/nonstd/value-ptr-light/include".
```
How do I fix this?  I tried to check, and I see that I at least have that directory.  I only have a folder named `nonstd` in there, though (in `/usr/local/jinja2cpp/thirdparty/nonstd/value-ptr-light/include`).  All of the other commands worked perfectly (I had to put `sudo` in front of each one, of course).
